dev/golangci/lint.go

```diff
@@ -54,7 +54,7 @@ func (run LintRun) Assert(ctx context.Context) error {
 		summaries []string
 	)
 	for _, iss := range issues {
-		if iss.Severity != "error" {
+		if !iss.IsError() {
```

Comment:
Does this actually fix the issue?

Reply:
Reworked this entirely, now the report is normalized right after parsing.